src/ArgParse.jl

```diff
@@ -1612,8 +1615,37 @@ function preparse(state::ParserState, settings::ArgParseSettings)
     end
 end
 
+# faithful reproduction of Python 3.5.1 argparse.py
+# partially Copyright © 2001-2016 Python Software Foundation; All Rights Reserved
+function read_args_from_files(arg_strings, prefixes)
+    new_arg_strings = AbstractString[]
+
+    for arg_string in arg_strings
+        if isempty(arg_string) || arg_string[1] ∉ prefixes
+            # for regular arguments, just add them back into the list
+            push!(new_arg_strings, arg_string)
+        else
+            # replace arguments referencing files with the file content
+            open(arg_string[2:end]) do args_file
```

Comment:
This will not work correctly if the "fromfile" character is Unicode (e.g. say that `arg_string` is `"∘test.args"`, then `arg_string[2:end]` would return an error). It should use `nextind(arg_string,1)` instead of `2`.